src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -655,13 +658,34 @@ private function addRouterSection(ArrayNodeDefinition $rootNode): void
     private function addSessionSection(ArrayNodeDefinition $rootNode): void
     {
         $rootNode
+            ->validate()
+                ->always(function (array $v): array {
+                    if ($v['session']['enabled']) {
+                        if (!\array_key_exists('cookie_secure', $v['session'])) {
+                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.cookie_secure" config option is deprecated. It will default to "auto" in 7.0.');
+                        }
+
+                        if (!\array_key_exists('cookie_samesite', $v['session'])) {
+                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.cookie_samesite" config option is deprecated. It will default to "lax" in 7.0.');
+                        }
+
+                        if (!\array_key_exists('handler_id', $v['session'])) {
+                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.handler_id" config option is deprecated. It will default to "session.handler.native_file" if "framework.session.save_path" is set or "null" otherwise in 7.0.');
```

Comment:
```suggestion
                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.handler_id" config option is deprecated. It will default to "session.handler.native_file" when "framework.session.save_path" is set or "null" otherwise in 7.0.');
```